config/parameters.json

```diff
@@ -4,14 +4,14 @@
       "host": "localhost",
       "port": 3306,
       "username": "root",
-      "password": "admin",
+      "password": "root",
```

Comment:
oh, i know test environment is not perfect for contributing, but next time please discard these changes